content/adventures/zh_Hans.yaml

```
@@ -258,8 +258,8 @@ adventures:
                 example_code: "## Example Hedy code\n```\nanswer = 0\nwhile answer != 25\n    answer = ask 'What is 5 times 5?'\nprint 'A correct answer has been given'\n```\n"
             16:
                 start_code: |-
-                    fruit = ['苹果', '香蕉', '樱桃']
-                    print fruit
+                    水果 = ['苹果', '香蕉', '樱桃']
+                    {print } 水果
```

```suggestion
                    {print} 水果
```
Space after print messes things up, I guess?